backend/routes/attendance.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const {
+  getWorkouts,
+  getWorkout,
+  createWorkout,
+  deleteWorkout,
+  updateWorkout
+} = require('../controllers/workoutController')
+const requireAuth = require('../middleware/requireAuth')
+
+const router = express.Router()
+
+//verifies that the user exists so they can do stuff
+router.use(requireAuth)
+//get all attendances
+router.get('/', getAttendance)
```

Comment:
can you call them something like /getattandance:id just to make it clearer as to which one is being used